gateway/1.1/modules/ROOT/pages/_partials/forward-proxy-config.adoc

```diff
@@ -0,0 +1,104 @@
+//tag::configuration[]
+A forward proxy, such as Squid, allows you to route all Flex Gateway connections through a single proxy connection. This allows you to isolate Flex Gateways to an internal network. 
+
+For both Connected Mode and Local Mode, you configure a forward proxy using YAML configuration files. 
+
+The following diagram demonstrates how an internal network communicates with an external network using a forward proxy. 
+
+image:forward-proxy-diagram.png[align=center]
+
+In the diagram, all communication from the internal Flex Gateway replicas travel through the forward proxy, then to Anypoint Platfrom. Communication from Anypoint Platform travels through the firewall first, then the forward proxy, before reaching the Flex Gateway replica.
+
+== Configuring Forward Proxy Parameters
+
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: Configuration
+metadata:
+  name: forward-proxy
+spec:
+  forwardProxy: 
+    https: // REQUIRED
+      address: http://proxy:8888 // REQUIRED
+      basicAuth: // OPTIONAL
+        username: <string> // OPTIONAL
+        Password: <string> // OPTIONAL
+    noProxy: // OPTIONAL
+       - fluentd // OPTIONAL
+
+----
+
+|===
+|Parameter | Required or Optional | Description
+
+| `https.address`
+| required
+| Proxy adddress
+
+| `basicAuth`
+| Optional
+| Supplies authentication username and password
+
+| `basicAuth.username`
+| Optional
+| Authentication username
+
+| `basicAuth.Password`
+| Optional
+| Authentication password
+
+| `noProxy`
+| Optional
+| Allows internal network connections to communicate without using the forward proxy
+|===
+
+
+
+== noProxy Parameter
+
+Adding the `noProxy` parameter allows services inside the trusted internal network to communicate without using the proxy.
+
+The following diagram demonstrates the differences in communication using noProxy:
+
+image:no-proxy-diagram.png[align=center]
+
+In the diagram, when noProxy is disabled, Flex Gateway communicates with the internal API endpoint using the forward proxy. When noProxy is enabled, Flex Gateway communicates directly with the internal API endpoint.
+
+== File Configuration Example
+
+. Create a YAML configuration file in the Flex Gateway configuration directory:
++
+[source,ssh]
+----
+sudo touch /usr/local/share/mulesoft/flex-gateway/conf.d/forward-proxy.yaml
+----
```

Comment:
This works for linux, but for the other targets a different method is used (in docker this file should be created in the mounted volume and in kubernetes with `kubectl apply`). For now I think we would want to use a similar structure as the other tutorials that create a `Configuration`, for example https://docs.mulesoft.com/gateway/latest/flex-local-shared-storage-config#configure-shared-storage-for-flex-gateway-as-a-kubernetes-ingress-controller